examples/manifests/consumer.yaml

```diff
@@ -15,15 +15,6 @@ spec:
       labels:
         app: consumer
     spec:
-      affinity:
```

Comment:
We need affinity so that the producer and consumer  deployed to a baremetal node  use label to target baremetal node 
oc label node <node>  app=local